### DISCUSSION TITLE
how to get locators color by using webdriverio +appium

### DISCUSSION THREAD

**Discussion by garyzhang21** [webdriverio/webdriverio#7617] [Q&A] [answered]
Hi，sorry
if i want to check one locator's color by getCSSProperty() and appium
but i can not find any parameters. so how to check the color? do you have ideas?

**Comment by christian-bromann** [comment, marked as answer]
What color? The text color or background color? Have you tried the [debug command](https://webdriver.io/docs/api/browser/debug/) to fiddle around with different options?
